client/me/billing-history/receipt.jsx

```diff
@@ -138,6 +139,7 @@ const BillingReceipt = React.createClass( {
 		return (
 			<div>
 				<DocumentHead title={ translate( 'Billing History' ) } />
+				<PageViewTracker path="/me/purchases/billing/:receiptId" title="Me > Billing History > Receipt" />
```

Comment:
I was deceived by the fact that current usages of `<PageViewTracker />` throughout Calypso use the `:paramName` syntax when there is a parameter. But in this case, it makes sense to keep the same path so we can continue tracking the same stat. Thanks for noticing :+1: